bme_install_persistant.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: deploy_node
+  name: Install and run tests
+  vars:
+    tempest_root: /opt/tempest_untagged/
+    test_name: persistent-resources-tests
+    test_type: persistent-clean
+
+  tasks:
+    - name: Clone git repo persistent-resources-tests
+      when: test_type == "persistent-clean"
+      command: /usr/local/bin/ansible utility_all -i /opt/openstack-ansible/playbooks/inventory -m shell -a "git clone https://github.com/osic/{{ test_name }}.git {{ tempest_root }}/{{ test_name }}"
+
+    - name: install persistent-resources-tests
+      command: /usr/local/bin/ansible utility_all -i /opt/openstack-ansible/playbooks/inventory -m shell -a "pip install {{ tempest_root}}/{{ test_name }}"
```

Comment:
Good point. Didn't consider that as the intention was starting from a bare environment and building up, or not have a long lasting environment due to the ease of a rebuild. But shouldn't hurt if the test is run multiple times between rebuilding.